installer.sh

```diff
@@ -666,6 +197,7 @@ install_necessary_packages() {
 		;;
 	esac
 }
+
 install_necessary_packages
```

Comment:
Can install_necessary_packages also be moved to the lib?

Reply:
yes, it can. i was undecided...